Jamfile

```diff
@@ -560,7 +560,7 @@ lib libiconv : : <name>iconv <link>shared <search>/usr/local/lib ;
 # openssl on linux/bsd/macos etc.
 lib gcrypt : : <name>gcrypt <link>shared <search>/opt/local/lib <search>/usr/local/lib : : <include>/opt/local/include <include>/usr/local/include ;
 lib z : : <link>shared <name>z <search>/usr/lib ;
-lib crypto : : <name>crypto <link>shared <search>/usr/lib <use>z : : <include>/opt/local/include <include>/usr/local/include ;
```

Comment:
Yes, it is to support cross-compilation, specially in macOS, which is the one having problems since openssl is part of the system. Now, I don't know if people expect the current behavior or how they use this. Would you consider merge this and cleanup more in a later step?